libs/detectors/x86/detector.py

```diff
@@ -1,3 +1,37 @@
+import json
+import os
+from pathlib import Path
+
+
+def get_or_create_model_json_file(config, device, camera_id):
+    """
+    If no model directory was created for a given device, the default values will be taken from the configuration file
+    and a model_<device>.json file will be created.
+    """
+    base_path = os.path.join(config.get_section_dict("App")["EntityConfigDirectory"], "sources", camera_id)
+    models_directory_path = os.path.join(base_path, "ml_models")
+    json_file_path = os.path.join(models_directory_path, f"model_{device}.json")
+
```

Comment:
The location and name of the custom models should be documented in the README.

Reply:
Done!